src/pages/progress_reports/SummarySection.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState } from 'react';
+import { FormField, FormFieldTypes } from 'src/components/forms/forms';
+import { Publication } from 'src/components/publications_list/Publication';
+import PublicationList from 'src/components/publications_list/PublicationList';
+import { FormFieldChange, FormState } from 'src/pages/progress_reports/ProgressReportFormState';
+
+interface SummarySectionProps {
+    readonly readOnly: boolean;
+    formState: FormState;
+    onFormChange: (newState: Partial<FormState>) => void;
+}
+
+export default function SummarySection(props: SummarySectionProps): React.JSX.Element {
+    const { readOnly, formState, onFormChange } = props;
+
+    const [publications, setPublications] = useState(formState.publications || []);
+    const [presentations, setPresentations] = useState(formState.presentations || []);
+
+    const onStateChange = (key: string, setState: React.Dispatch<Publication[]>, publications: Publication[]) => {
+        onFormChange({ [key]: publications });
+        setState(publications);
+    };
+
+    const onPublicationChange = (publications: Publication[]) => {
+        onStateChange('publications', setPublications, publications);
+    }
+
+    const onPresentationChange = (presentations: Publication[]) => {
+        onStateChange('presentations', setPresentations, presentations);
+    }
+
+    return (
+        <div data-cy='summary-section'>
+            <div className='progress-report-step-card'>
+                {readOnly ? <h3>Review a Progress Report</h3> : <h3>Step 1: Submit a Progress Report</h3>}
+
+                <div className='progress-report-row'>
+                    <FormField
+                        id='progressReportSummary'
+                        type={FormFieldTypes.TEXTAREA}
+                        title='1.1 Summary of Progress'
+                        description='Please summarize your research on this project since your initial request or most recent renewal in the space below. Please describe whether and how the dataset(s) was used, including referencing the dataset(s) by name in your summary.'
+                        placeholder='Please provide an update here.'
+                        rows={6}
+                        maxLength={2200}
```

Comment:
It would be nice to avoid a magic number here, in case this value changes in the future. Also I see `2200` in many other places in the UI code, so this appears to be an existing magic number.